cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -297,38 +473,26 @@ DO NOT: Tell someone if the account exists on LogOn, Registration or Password re
 
 The feedback to the user should be identical whether or not the account exists, both in terms of content and behaviour: e.g. if the response takes 50% longer when the account is real then membership information can be guessed and tested.
 
-## A3 Cross Site Scripting
+### Missing function-level access control
```

Comment:
It miss the info link to the [Access Control CS](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Access_Control_Cheat_Sheet.md) and the [Authorization CS](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Authorization_Testing_Automation.md) like for the LDAP injection.

Reply:
valid point